deploy/deploy.sh

```diff
@@ -29,6 +30,10 @@ else
   exit 1
 fi
 
+echo "Upgrading bonfire..."
+pip install --upgrade crc-bonfire
```

Comment:
this should be a part of readme and we should upgrade dependent commands here

Reply:
You can try to get `bonfire version` and if its less than the minimum required version, you can throw the help in console asking the user to upgrade and exit the script

Reply:
I added your suggestion ;)